code/game/objects/items/weapons/twohanded.dm

```diff
@@ -546,9 +541,7 @@ obj/item/weapon/twohanded/
 /obj/item/weapon/twohanded/knighthammer
 	name = "singuloth knight's hammer"
 	desc = "A hammer made of sturdy metal with a golden skull adorned with wings on either side of the head. <br>This weapon causes devastating damage to those it hits due to a power field sustained by a mini-singularity inside of the hammer."
-
-	icon_override = 'icons/mob/in-hand/swords.dmi'
-	icon_state = "adrhammer0"
+	icon_state = "knighthammer0"

```

Comment:
..... :c 

Whatever, Im not gonna defend it. It was fun while it lasted. And no, i never asked or implemented that, if you were wondering.